libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java

```diff
@@ -1050,6 +1050,65 @@ private AnalyticsTrackerMixpanelInstructionsForStat instructionsForStat(
                 instructions = AnalyticsTrackerMixpanelInstructionsForStat.
                         mixpanelInstructionsForEventName("Login - Auto Fill Credentials Updated");
                 break;
+            case LOGIN_PROLOGUE_VIEWED:instructions = AnalyticsTrackerMixpanelInstructionsForStat.
```

Comment:
Hey @aerych , quite minor but, let's break this line and start the `instructions = ..` in the next line, like the rest of the cases you've already added. Thanks!